src/mqt/bench/qiskit_helper.py

```diff
@@ -22,30 +22,54 @@ def get_native_gates(gate_set_name: str):
     raise ValueError("Unknown gate set name: " + gate_set_name)
 
 
-def get_ibm_native_gates():
+def get_ibm_native_gates() -> list[str]:
     return ["rz", "sx", "x", "cx", "measure"]
 
 
-def get_rigetti_native_gates():
+def get_rigetti_native_gates() -> list[str]:
     return ["rx", "rz", "cz", "measure"]
 
 
-def get_ionq_native_gates():
+def get_ionq_native_gates() -> list[str]:
     return ["rxx", "rz", "ry", "rx", "measure"]
 
 
-def get_oqc_native_gates():
+def get_oqc_native_gates() -> list[str]:
     return ["rz", "sx", "x", "ecr", "measure"]
 
 
+@overload
 def get_indep_level(
     qc: QuantumCircuit,
-    num_qubits: int,
+    num_qubits: int | None,
+    file_precheck: bool,
+    return_qc: Literal[True] = True,
```

Comment:
```suggestion
    return_qc: Literal[True],
```
might be wrong, but I do not think, that this argument is necessary